web/themes/custom/usagov/templates/field--node--field-custom-twig-content-reporte-problemas-en-este-sitio-web.html.twig

```diff
@@ -144,5 +120,5 @@
 <p class="clearfix"></p>
 
 <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.6.4/jquery.min.js"></script>
-<script src="https://www.google.com/recaptcha/api.js?hl=es"></script>
+<script src="https://www.google.com/recaptcha/api.js"></script>
```

Comment:
Maybe we still need the ?hl=es query string. If you didn't already check, please take a look at whether the recaptcha badge has text in English on Spanish pages, and if so, if putting the query string back on changes it to Spanish.

Reply:
Following up, it looks like we do want the query string. It causes the text on the recaptcha to come through in Spanish.